content/javascript/concepts/dates/dates.md

```diff
@@ -0,0 +1,131 @@
+---
+Title: "Dates" 
```

Comment:
Amazing. Love that you changed "Dates in JavaScript" to just "Dates" without even me telling you!